openapi.json

```diff
@@ -266,11 +266,11 @@
         ]
       }
     },
-    "/organizations/{orgId}/clusters/{clusterId}/users/{userId}/report/{ruleID}": {
+    "/organizations/{orgId}/clusters/{clusterId}/users/{userId}/rules/{ruleId}/report": {
```

Comment:
Why this was changed? As I see it's against RestAPI, because I want to get rule, which is part of report, so I use ID.

Reply:
Because it's used everywhere https://platform-docs.cloud.paas.psi.redhat.com/rest/urls.html  and your version doesn't make any sense. Why do we use this convention for other keywords(oraganizations, clusters, users) then?

Reply:
This version according to docs makes even less sense, because `report` is already array of rules, so if I want certain rule, I'll just add Id of rule and that's it

Reply:
No, you're passing id of a rule, not id of a report

Reply:
btw, we could also use just `/organizations/{orgId}/clusters/{clusterId}/users/{userId}/rules/{ruleId}/`

Reply:
Logic of this is `report==rules`, report is just bunch of rules, that's why `id` is about rule not about report.

Reply:
It just introduces a lot of confusions, IMO

Reply:
Ok, let it be `/organizations/{orgId}/clusters/{clusterId}/users/{userId}/rules/{ruleId}`

Reply:
done